gradle-plugin/kubernetes/src/test/java/org/eclipse/jkube/gradle/plugin/GradleUtilTest.java

```diff
@@ -76,12 +76,28 @@ public void extractProperties_withComplexMap_shouldReturnValidProperties() {
     final JavaProject result = convertGradleProject(project);
     // Then
     assertThat(result.getProperties())
-        .hasSize(2)
+        .isNotEmpty()
```

Comment:
You can use our SystemMock class to deal with this so we have a predictable size and values

Reply:
SystemMock seems to be using JMockit for mocking:

https://github.com/eclipse/jkube/blob/6afbb4f371cec30e90b895490ccdd060ec53503b/jkube-kit/common-test/src/main/java/org/eclipse/jkube/kit/common/SystemMock.java#L19-L22

I'm not sure how to proceed since we already have plans to replace JMockit by Mockito in future. Shall I add a todo here for now to refactor this once https://github.com/eclipse/jkube/issues/857 is resolved?

Reply:
Let's keep it this way. Once we merge this branch we can complete #958